torchci/lib/JobClassifierUtil.ts

```diff
@@ -2,6 +2,41 @@ import { GroupedJobStatus, JobStatus } from "components/job/GroupJobConclusion";
 import { getOpenUnstableIssues } from "lib/jobUtils";
 import { IssueData, RowData } from "./types";
 
+type RepoViableStrictBlockingJobsMap = {
+  [key: string]: RegExp[];
+};
+
+// Source of truth for these jobs is in https://github.com/pytorch/pytorch/blob/main/.github/workflows/update-viablestrict.yml#L26
+export const VIABLE_STRICT_BLOCKING_JOBS: RepoViableStrictBlockingJobsMap = {
+  "pytorch/pytorch": [/pull/i, /trunk/i, /lint/i, /linux-aarch64/i],
```

Comment:
This might be hard to maintain.

I'm curious if we can have this be populated somewhere in clickhouse and we can just pull it from there.

It's fine for a first pass but we should make a TODO / issue about revising it to do the right thing